### PR TITLE
Creating new plugin tutorial

### PR DESCRIPTION
**Pull request by shuntrho**
Added short guide on creating new Membrane plugin.

### CODE REVIEW

File: create_new_plugin/create_new_plugin.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Creating your own plugin
+
+During the development of Membrane Framework we aim at designing fewer, but higher quality plugins. However we also kept extendability and reusability in mind. That's why it is easy for developers like you to create their own custom plugin, which satisfies their needs.
+
+In this short guide we provide you with an overview of how to create your own Membrane plugin and how to integrate it into your project.
+
+# Membrane plugin template
+
+In order to create new plugin we recommended using the [template](https://github.com/membraneframework/membrane_template_plugin) which has been made for this very purpose and which will be the base of your plugin.
```
Comment (varsill) [inline create_new_plugin/create_new_plugin.md:9]:
```suggestion
In order to create a new plugin, we recommend using the [template](https://github.com/membraneframework/membrane_template_plugin) which has been made for this very purpose and which will be the base of your plugin.
```

File: create_new_plugin/create_new_plugin.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Creating your own plugin
+
+During the development of Membrane Framework we aim at designing fewer, but higher quality plugins. However we also kept extendability and reusability in mind. That's why it is easy for developers like you to create their own custom plugin, which satisfies their needs.
```
Comment (varsill) [inline create_new_plugin/create_new_plugin.md:3]:
```suggestion
During the development of Membrane Framework we aim at designing fewer, but higher quality plugins. However, we also kept extendability and reusability in mind. That's why it is easy for developers like you to create their own custom plugin, which satisfies their needs.
```

**Review by varsill — CHANGES_REQUESTED** [review]
Great, that's what we had on our mind! :D 
Just these two fixes concerning the grammar are required

**Review by varsill — APPROVED** [review]
LGTM! 🥇

**Comment by varsill** [comment]
We should think if it would be good to add a link to the new tutorial in the README.md here and there: https://github.com/membraneframework/membraneframework.github.io/blob/master/index.md
therefore making it public

**Comment by shuntrho** [comment]
I wonder if it should be added in the plugin repo as well. https://github.com/membraneframework/membrane_template_plugin

**Comment by varsill** [comment]
Hmm, as far as I know, when repo is marked as a template, it should contain only necessary stuff as it will be available in somebody's project - we cannot separate the readme out of the template :/ 
Another thing is that we should finally make template public

**Pull request merged**